### PR TITLE
fix typo (resx and man files)

### PR DESCRIPTION
**Pull request by spaette**
src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man
src/System.Management.Automation/resources/EventResource.resx

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

File: src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man [hunk 1]
```diff
@@ -4869,7 +4869,7 @@
             />
         <string
             id="PS_PROVIDER.event.E_A_CREATE_SERVER_REMOTESESSION.message"
-            value="Request %1. Creating a server remote session. UserName: %2 Custome Shell Id: %3"
```
Comment (TravisEz13) [inline src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man:4872]: Please open a PR in the PowerShell-Native repo for this issue.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
See comment

File: src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man (hunk 1 above)
Comment (spaette) [reply src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man:4872]: Should the change to the .man file in this PR be reverted?

**Review by TravisEz13 — APPROVED** [review]

File: src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man (hunk 1 above)
Comment (TravisEz13) [reply src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man:4872]: No need to revert.  I'm not sure if the manifest file here is still needed, but it is not the one used by the installer.

**Comment by TravisEz13** [comment]
Thanks for the fixes.

**Pull request merged**